Bottom: 4b9c00872f04d4b6f88d4006f53a90777eafcc33
Top:    8cf1fe7e564f969740ade3952b079bcb521d9c2d
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 14:43:34 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 25f8968..57a136b 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -590,6 +590,7 @@ int rpc_portmap_getclient(struct conn_info *info,
 	info->program = rpc_getrpcbyname(rpcb_prog);
 	info->port = rpc_getrpcbport(proto);
 	info->version = rpcb_version;
+	error(LOGOPT_ANY, "program %d port %d version %d", info->program, info->port, info->version);
 	info->proto = proto;
 	info->send_sz = RPCSMALLMSGSIZE;
 	info->recv_sz = RPCSMALLMSGSIZE;
@@ -635,6 +636,7 @@ int rpc_portmap_getport(struct conn_info *info,
 		pmap_info.port = rpc_getrpcbport(info->proto);
 		pmap_info.program = rpc_getrpcbyname(rpcb_prog);
 		pmap_info.version = rpcb_version;
+		error(LOGOPT_ANY, "program %d port %d version %d", pmap_info->program, pmap_info->port, pmap_info->version);
 		pmap_info.proto = info->proto;
 		pmap_info.send_sz = RPCSMALLMSGSIZE;
 		pmap_info.recv_sz = RPCSMALLMSGSIZE;
